framework/web/XmlResponseFormatter.php

```diff
@@ -37,7 +37,7 @@ class XmlResponseFormatter extends Component implements ResponseFormatterInterfa
      */
     public $encoding;
     /**
-     * @var string the name of the root element.
+     * @var string the name of the root element or null if no root tag should be added.
```

Comment:
in Html helper we use false for no tag. In general we could also allow it to be empty(). as `<>` would not be a valid xml tag anyway.